MediaGalleryUi/Console/Command/IndexAssets.php

```diff
@@ -51,19 +51,32 @@ public function __construct(
     protected function configure()
     {
         $this->setName('media-gallery:index');
-        $this->setDescription('Scan media directory for media gallery asset and write their parameters to database');
+        $this->setDescription(
+            'Run media gallery ui indexers.'
+        );
     }
 
     /**
      * @inheritdoc
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $output->writeln('Uploading assets information from media directory to database...');
-        $this->state->emulateAreaCode(Area::AREA_ADMINHTML, function () {
-            $this->imagesIndexer->execute();
+        $this->state->emulateAreaCode(Area::AREA_ADMINHTML, function () use ($output) {
+            foreach ($this->indexers as $indexer) {
+                /** @var ImagesIndexerInterface $indexer */
+                if ($indexer instanceof ImagesIndexerInterface) {
+                    $indexTitle = __($indexer->getTitle() . ' index ');
```

Comment:
```suggestion
                    $indexTitle = __('%1 index ', $indexer->getTitle());
```

Reply:
@hws47a thank you for pointing this part of a code. I made adjustments. Please, review :)